go/hyperkit.go

```diff
@@ -353,6 +353,12 @@ func (h *HyperKit) String() string {
 	return string(s)
 }
 
+// Command builds and returns a single string of the command line
+func (h *HyperKit) Command(cmdline string) string {
+	h.buildArgs(cmdline)
+	return h.CmdLine
```

Comment:
I'm not sure that `h.CmdLine` is actually usable for what you need -- it has collapsed the individual elements of `argv` into a single string (without any quoting) and it's not entirely clear that you can always reverse that..